lib/plugins/nuxtentBodyComponent.template.js

```diff
@@ -3,25 +3,29 @@ import Vue from 'vue'
 const mdComps = {}
 
 function importAllMdComps (r) {
-  r.keys().forEach(key => (mdComps[key] = r(key)))
+  r.keys().forEach(key => (mdComps[key] = r(key).default))
 }
 
 importAllMdComps(
   require.context(<%= JSON.stringify(options.srcDirFromPlugin) %>, true, /\.comp\.md$/)
 )
 
 Vue.component('nuxtent-body', {
-  functional: true,
-  props: {
-    body: { required: true }
-  },
-  render (h, ctx) {
-    const { body } = ctx.props
-    if (typeof body === 'object') {
-      const MarkdownComponent = mdComps[body.relativePath]
-      return <MarkdownComponent />
+  render: function(h) {
```

Comment:
I changed this back to regular JSX syntax since it is easier to read. I tested both versions and neither provides an advantage over the other.